Dockerfile

```diff
@@ -1,31 +1,17 @@
-FROM ubuntu as build-hapi
-
-ENV PATH="/tmp/apache-maven-3.6.0/bin:${PATH}"
-
-ENV TZ=Etc/UTC
-RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
-
-WORKDIR /tmp
-RUN apt-get update && \
-    apt-get install git -y && \
-    apt-get install sed -y && \
-    apt-get install wget -y && \
-    apt-get install openjdk-11-jdk -y
-
-RUN wget https://downloads.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz && tar xzf apache-maven-3.6.3-bin.tar.gz
-RUN export PATH=/tmp/apache-maven-3.6.3/bin:${PATH}
-
+FROM maven:3.6.3-jdk-11-slim as build-hapi
 WORKDIR /tmp/hapi-fhir-jpaserver-starter
 
-COPY . .
+COPY pom.xml .
+RUN mvn dependency:go-offline
 
-RUN /tmp/apache-maven-3.6.3/bin/mvn clean install -DskipTests
+COPY src/ /tmp/hapi-fhir-jpaserver-starter/src/
+RUN mvn clean install -DskipTests
```

Comment:
Don't you think that if the tests fail the build should fail?

Reply:
I optimized the dockerfile - as can be seen, it ran with '-DskipTests' before. As such I haven't changed the semantics with regards to the use of maven

Reply:
FYI @jvitrifork and @jkiddo is the same person